.github/workflows/checks-codecov.yaml

```diff
@@ -69,22 +69,29 @@ jobs:
       - name: Generate
         run: make generate
 
+      - name: Generate CRD manifests
+        run: make crd-generate
+
       # Possibly useful for debugging if the next step produces an error
       - name: Show diff after `make generate`
         run: git diff --stat --patch
 
       - name: Check for uncommitted changes
         run: |
-          if ! git diff --exit-code -s; then
-            for f in $(git diff --exit-code --name-only); do
-              echo "::error file=$f,line=1,col=1,endColumn=1::File was modified in build"
-            done
+          git status --porcelain
```

Comment:
I see this replaces the existing check. Is that ok, and should we now run `make all` when this check is triggered?